EasyPost.Tests/cassettes/report/create_payment_log_report.json

```diff
@@ -1,50 +1,51 @@
 [
   {
+    "RecordedAt": "2022-03-23T14:55:25.084273-06:00",
     "Request": {
+      "Body": "{\"start_date\":\"2022-02-01\",\"end_date\":\"2022-02-03\"}",
+      "ContentHeaders": {
+        "Content-Type": "application/json; charset=utf-8",
+        "Content-Length": "51"
+      },
       "Method": "POST",
-      "URI": "https://api.easypost.com/v2/reports/payment_log?start_date=2022-02-01&end_date=2022-02-03",
       "RequestHeaders": {
         "Accept": "application/json",
         "Authorization": "********",
         "content_type": "application/json",
         "User-Agent": "EasyPost/v2,CSharpClient/2.8.1,.NET/5.0.13"
```

Comment:
Shouldn't this user agent say `2.8.2`? Why is it not using the most recent version? Leads me to believe that the header bug fix never made it to the master branch (this is why we must keep all of these things in separate PRs, pulling them into an already existing one leads to this confusion and if done wrong could lead to certain changes getting missed).

Reply:
I'll merge that in first